src/main/java/seedu/duke/reflection/SelfReflection.java

```diff
@@ -0,0 +1,89 @@
+package seedu.duke.reflection;
+
+import java.util.ArrayList;
+
+/**
+ * The main body of self reflect section.
+ */
+public class SelfReflection {
+    private static final String LOGO =
+            "  _____ ______ _      ______   _____  ______ ______ _      ______ _____ _______ _____ ____  _   _ \n"
+            + " / ____|  ____| |    |  ____| |  __ \\|  ____|  ____| |    |  ____/ ____|__   __|_   _/ __ \\| \\ | |\n"
+            + "| (___ | |__  | |    | |__    | |__) | |__  | |__  | |    | |__ | |       | |    | || |  | |  \\| |\n"
+            + " \\___ \\|  __| | |    |  __|   |  _  /|  __| |  __| | |    |  __|| |       | |    | || |  | | . ` |\n"
+            + " ____) | |____| |____| |      | | \\ \\| |____| |    | |____| |___| |____   | |   _| || |__| | |\\  |\n"
+            + "|_____/|______|______|_|      |_|  \\_\\______|_|    |______|______\\_____|  |_|  "
+            + "|_____\\____/|_| \\_|\n";
+    private static final String GREETING_MESSAGE = "Welcome to WellNUS++ Self Reflection section:D"
+            + System.lineSeparator() + "Feel very occupied and cannot find time to self reflect?"
+            + System.lineSeparator() + "No worries, this section will give you the opportunity to reflect "
+            + "and improve on yourself!!";
+
+    // Questions are adopted from website: https://www.usa.edu/blog/self-discovery-questions/
```

Comment:
Good to link the source!

Reply:
u mean links have a specific format in JavaDoc?